flutter_vlc_player_platform_interface/lib/src/utils/options/vlc_subtitle_options.dart

```diff
@@ -0,0 +1,99 @@
+import '../../enums/subtitles/vlc_subtitle_text_direction.dart';
+import '../../enums/subtitles/vlc_subtitle_color.dart';
+import '../../enums/subtitles/vlc_subtitle_thickness.dart';
+
+class VlcSubtitleOptions {
+  final List<String> options;
+
+  VlcSubtitleOptions(this.options);
+
+  /// Set subtitle font (must be accessible by vlc)
+  static String font(String font) {
+    return '--freetype-font=$font';
+  }
+
+  /// Set subtitle mono font (must be accessible by vlc)
+  static String monofont(String monofont) {
+    return '--freetype-monofont=$monofont';
+  }
+
+  /// Set subtitle font size
+  static String fontSize(int size) {
+    return '--freetype-fontsize=$size';
+  }
+
+  /// Set subtitle rel font size
+  static String relFontSize(int size) {
+    return '--freetype-rel-fontsize=$size';
+  }
```

Comment:
Let's use full naming where we can. I've had to Google it to verify that it's a _relative_ font size setting :).
```suggestion
  /// Set subtitle relative font size
  static String relativeFontSize(int size) {
    return '--freetype-rel-fontsize=$size';
  }
```